aubreylib/resource.py

```diff
@@ -215,7 +218,7 @@ def get_metadata_file(self, parsed_mets):
         else:
             raise ResourceObjectException("Descriptive metadata not defined " +
                                           "in the METS file.")
-        for attribs, value in mdRef.attrib.items():
+        for attribs, value in list(mdRef.attrib.items()):
```

Comment:
I am not sure we need this. I tried locally and even though the data type is different in 3.7 for this (dict_items vs list), we can still iterate over it the same way.